design/019-local_development_flow.md

```diff
@@ -13,8 +13,8 @@
 The so called Conan “local flow”, composed by the commands ``source``, ``build``, ``package`` will be simplified to the following:
 
 - The main local development flow will be just ``conan install`` plus using the native build tools. The already existing ``generate()`` method, together with the ongoing work in the ``conan.tools.xxx`` new helpers like ``CMakeDeps`` and ``CMakeToolchain`` should generate all the files and information necessary to build, without requiring Conan or the ``conan build`` command.
-- Remove the ``conan package`` command, its functionality is fully covered by the ``conan export-pkg`` command
-- The ``conan build`` command might fully disappear if the above doesn’t have any exception. If there are some flows that cannot be fully fulfilled with a pure generator approach, a local ``conan build`` might be useful. If that is the case, the command will be complete, computing a full dependency graph that doesn’t rely on the previously saved state.
+- Remove the ``conan package`` command, its functionality is fully covered by the ``conan export-pkg`` command.
+- The ``conan build`` command will remain as a high level command, useful for developers and CI. The ``conan build`` command will be complete, computing a full dependency graph that doesn’t rely on the previously saved state, being equivalent to doing a ``conan install`` followed to a call to the ``build()`` recipe method.
```

Comment:
what about `source`? does install followed by build work without the source?

Reply:
There is a note above, clarifying that ``source`` is still not known. I have explicitly left this unspecified, we want to check the possible use cases more calmly before defining this.